Samples/office-add-in-commands/auto-open-task-pane/README.md

```diff
@@ -0,0 +1,171 @@
+---
+page_type: sample
+urlFragment: word-auto-open-with-document
+products:
+  - office-add-ins
+  - office-word
+  - office
+  - office-365
+languages:
+  - javascript
+extensions:
+  contentType: samples
+  technologies:
+    - Add-ins
+  createdDate: '05/03/2023 10:00:00 AM'
+description: "Configure a document to automatically open your Office Add-in's task pane when the document is opened."
+---
+
+# Auto-open a task pane with a document
+
+## Summary
+
+Some scenarios for your Office Add-in may require the task pane to automatically open with certain documents. This sample shows how to configure a document to automatically open your task pane when the document is opened by a user. The auto-open task pane feature requires the [AddInCommands 1.1](https://learn.microsoft.com/javascript/api/requirement-sets/common/add-in-commands-requirement-sets) requirement set.
+
+![The autoopen sample's task pane with buttons to turn autoopen on and off.](./images/auto-open-sample.png)
+
+## Features
+
+- Shows how to configure a document to auto-open when the document is opened.
+- Shows how to turn off auto-open for the document.
+
+## Applies to
+
+- Word on Windows, Mac, and in a browser.
+
+## Prerequisites
+
+- Microsoft 365 - Get a [free developer sandbox](https://developer.microsoft.com/microsoft-365/dev-program#Subscription) that provides a renewable 90-day Microsoft 365 E5 developer subscription.
+
+## Run the sample on Word on the web
+
+This sample is hosted directly from this GitHub repo. Use the following steps to sideload the manifest.xml file to see the sample run.
+
+1. Download the **manifest.xml** file from the folder for this sample.
+1. Open [Office on the web](https://office.com/).
+1. Choose **Word**, and then open a new document.
```

Comment:
Does typing `word.new` in a browser work outside of Microsoft/Edge? I've never tried...

Reply:
I'm not sure but I think we stick to the UI in the instructions when possible.